ape_safe/_cli.py

```diff
@@ -59,61 +59,96 @@ def _list(cli_ctx, network):
 @click.argument("address", type=AddressType)
 @non_existing_alias_argument()
 def add(cli_ctx, network, address, alias):
-    address = convert(address, AddressType)
-    safe_contract = Contract(address)
+    _ = network  # Needed for NetworkBoundCommand
+    address = cli_ctx.conversion_manager.convert(address, AddressType)
+    safe_contract = cli_ctx.chain_manager.contracts.instance_at(address)
     version_display = safe_contract.VERSION()
-    req_confs = safe_contract.getThreshold()
+    required_confirmations = safe_contract.getThreshold()
     signers_display = "\n    - ".join(safe_contract.getOwners())
 
     cli_ctx.logger.info(
         f"""Safe Found
     network: {network}
     address: {safe_contract.address}
     version: {version_display}
-    required confirmations: {req_confs}
+    required confirmations: {required_confirmations}
     signers:
     - {signers_display}
     """
     )
 
     if click.confirm("Add safe"):
-        accounts.containers["safe"].save_account(alias, address)
+        container = cli_ctx.account_manager.containers["safe"]
```

Comment:
**thought**: We may want to just grab the container at the top of the module once and handle the `IndexError` like 
 `ERROR: ape-safe plugin failed to load!`.

**reason**: I am thinking about a scenario where (in development) the safe account plugin fails to load (causing `IndexError` here), but the CLI is still registered, allowing you to type commands. And the error may be confusing.

Reply:
fixed